src/lib.rs

```diff
@@ -2022,6 +2022,309 @@ fn digraph_dijkstra_shortest_path_lengths(
     })
 }
 
+fn _all_pairs_dijkstra_path_lengths<Ty: EdgeType + Sync>(
+    py: Python,
+    graph: &StableGraph<PyObject, PyObject, Ty>,
+    edge_cost_fn: PyObject,
+) -> PyResult<AllPairsPathLengthMapping> {
+    if graph.node_count() == 0 {
+        return Ok(AllPairsPathLengthMapping {
+            path_lengths: HashMap::new(),
```

Comment:
Wouldn't `AllPairsPathLenghtMapping::new()` do the same thing?

Reply:
It would, This was less typing :) and also consistent with how we build the other returns.

Reply:
That makes sense, thanks for clarifying it